source/developers/concepts/platform_overview.rst

```diff
@@ -0,0 +1,83 @@
+Platform Overview
+#################
+
+The Open edX Platform is made up of multiple web services that work together to
+provide all the tools you might need to build and run courses online.
+
+Tech Stack
+**********
+
+While there are a few exceptions for the most part the technology stack for the
+Open edX Platform is based on `Django`_ based backend servers and `React`_ based
+frontend apps.  There are multiple frontend apps which are usually referred to as
+MFEs(Micro-FrontEnds) within the ecosystem and community.
+
+The core code base lives on GitHub in the `openedx`_ organization.
+
+Platform Layout
+***************
+
+While the platform contains many services, MFEs and libraries the core of the
+platform lives in the `edx-platform`_ and `frontend-platform` repositories.
+
+* edx-platform - This was the first service created in the Open edX ecosystem and
+  contains the legacy monolith application. That application currently can run
+  in two modes.
+
+  As the LMS(Learning Management System) it serves as the backend
+  for learner data as they go through courses, programs, and serves some legacy
+  UI elements.
+
+  As the CMS(Content Management System) it serves as the place where instructors
```

Comment:
For clarity I think it should specify that the name of the authoring environment is Studio.